edb/server/protocol/protocol.pyx

```diff
@@ -175,5 +209,33 @@ cdef class HttpProtocol:
         else:
             self.resume()
 
-    async def handle_request(self, request, response):
-        raise NotImplementedError
+    async def handle_request(self, HttpRequest request, HttpResponse response):
+        path = urllib.parse.unquote(request.url.path.decode('ascii'))
+        path = path.strip('/')
+        path_parts = path.split('/')
+
+        if len(path_parts) >= 3:
+            root, dbname, extname, *args = path_parts
+            db = self.server.get_db(dbname=dbname)
+
+            if root == 'db':
+                if extname == 'graphql':
```

Comment:
I assume we will use some extension registry here instead of hardcoding, right?

Reply:
Yea, once we have it.